### PR TITLE
added community council / enforcement

### PR DESCRIPTION
**Pull request by heartsucker**
Answers the questions:

- What is the Community Council?
- How do I submit a report?
- What happens when I submit?
- How does the CC mitigate harm?

### CODE REVIEW

**Comment by ghost** [comment]
I would prefer a name that does not suggest it is more than dealing with the CoC. Like "CoC contacts".

**Comment by heartsucker** [comment]
I think that the idea of a CoC and community backwards. The CoC is just a codification of the ideal we hold as a community. Without a CoC we would still be self regulating, but the CoC just spells it out as way to communicate this to new people. The people we nominated to be on community team would have done moderating and arbitration already, so they're not really CoC contacts. I think calling them CoC Contacts puts the document first and in an elevated position.

At conference and other events, people who are points of contact for disputes and harm reports are called awareness teams, but I don't think this quite fits. Community Team maybe?

File: CoC.md [hunk 1]
```diff
@@ -114,6 +114,77 @@ following behaviors:
   any other violation of this code. That includes trolling and other forms of
   consistently disruptive behavior.
 
+## Enforcement
+
+We do not tolerate unacceptable behavior from any community member, and there
+are no exceptions for those in positions of power such as maintainers,
+sponsors, funders, or other individuals withe decision making authority.
```
Comment (eloquence) [inline CoC.md]: typo: withe->with

File: CoC.md [hunk 2]
```diff
@@ -114,6 +114,77 @@ following behaviors:
...
+Further, people in positions of power can wield it to exacerbate the effects of
+harassment and to diminish the repercussions, those who are informal or formal
```
Comment (eloquence) [inline CoC.md]: I would suggest: "and to diminish the repercussions. For these reasons, those who are informal .."

File: CoC.md [hunk 3]
```diff
@@ -114,6 +114,77 @@ following behaviors:
...
+You can contact the whole Community Coucil or members individually.
+
+You should contact the Community Council if you have questions or concerns about
+the Code of Conduct (including improvements) or if you feel that you have
+witnessed a Code of Conduct violation. In the even of a violation either
+directed at yourself or someone else, please contact the Community Council as
+soon as possible through whatever analog or digital medium you are most
+comfortable with.
+
+Should report be about any member of the Community Council or if there is a
```
Comment (eloquence) [inline CoC.md]: Missing word. "Should report" -> "Should your report"

File: CoC.md [hunk 4]
```diff
@@ -114,6 +114,77 @@ following behaviors:
...
+conflict of interest, that member will recuse themselves from the conflict
+resolution proceedings. They will not be involved with the discussion,
+documentation, communications, or decisions made by the rest of the Community
+Council in regards to the incident.
```
Comment (eloquence) [inline CoC.md]: "in regards to" or "with regard to". See, e.g., https://brians.wsu.edu/2016/05/24/in-regards-to/

File: CoC.md [hunk 5]
```diff
@@ -114,6 +114,77 @@ following behaviors:
...
+person(s) who have been harmed or are at risk of harm and the reporter(s). No
+actions will be taken without the consent of the person who has been harmed or
+is at risk of harm except in cases where danger or harm are imminent.
+
+All reports to the Community Council, no matter how minor or severe, will be
+taken seriously and looked into.
+
+## How the Community Council responds to incidents
+
+The Community Council does not have a fixed set responses to some enumerated set
```
Comment (eloquence) [inline CoC.md]: missing word: "fixed set responses" -> "fixed set of responses"

File: CoC.md [hunk 6]
```diff
@@ -114,6 +114,77 @@ following behaviors:
...
+of incidents that may occur. The Community Council operates on a case-by-case
+basis taking into account past behavior; the relationship between the person(s)
+who were harmed and the person(s) causing the harm; the responses of the
+person(s) who caused harm; and the perceived threat of future harm.
+
+Actions the Community Council may take to mitigate harm includes, but is not
```
Comment (eloquence) [inline CoC.md]: "includes, but is not"->"include, but are not"

File: CoC.md [hunk 7]
```diff
@@ -114,6 +114,77 @@ following behaviors:
...
+limited to:
+
+- A simple warning
+- A temporary ban from email lists, chat channels, repositories, or other online
+  communication mediums
+- Informal mediation
```
Comment (eloquence) [inline CoC.md]: I would move this to or near the top of the list, as the list generally seems to imply escalating consequences.

**Comment by eloquence** [comment]
This looks great, @heartsucker! Left some minor nits in the comments.

Regarding the naming, I think @dachary's main concern is that the scope is clear from the name. "Community Moderation and Safety Team" or something along those lines might work?

**Comment by ghost** [comment]
> Community Moderation and Safety Team

That would work. Teams do no imply a hierarchy, they are a collection of individuals with a common interest and commitment. They do not have any power delegation or power over anyone.

File: CoC.md [hunk 8]
```diff
@@ -114,6 +114,77 @@ following behaviors:
...
+
+### How to get help
+
+The Community Council is made up established members of the community who
+assist with resolving conflicts within the community.
+
+Currently the Community Council consists of:
+
+- Jen Helsby (`@redshiftzero`) - Lead Engineer - [jen@freedom.press](mailto:jen@freedom.press)
+- Loic Dachary (`@dachary`) - Maintainer - [loic@freedom.press](mailto:loic@freedom.press)
```
Comment (ghost) [inline CoC.md]: `loic@dachary.org`

File: CoC.md [hunk 9]
```diff
@@ -114,6 +114,77 @@ following behaviors:
...
+- A simple warning
+- A temporary ban from email lists, chat channels, repositories, or other online
+  communication mediums
+- Informal mediation
+- A temporary ban from events or community spaces
+- Permanent expulsion from the community
+
+Once the Community Council has reached a decision on how to mitigate the harm or
+risk of harm, the person(s) on the receiving end of the mitigation(s) may appeal
+the decision by writing or otherwise communicating with the Community Council.
```
Comment (ghost) [inline CoC.md]: The rationale leading the [contacts] to the conclusion will be clearly explained to everyone involved.

Note: people who turn to the point of contact expect a ruling. But they also expect a ruling that they can understand. They may disagree but they should never be in a position to critize a decision as arbitrary or defying common sense.

Comment (heartsucker) [reply CoC.md]: I'm not sure I follow you here.

**Review by redshiftzero — APPROVED** [review]

**Pull request merged**